examples/deadline/All-In-AWS-Infrastructure-SEP/python/README.md

```diff
@@ -0,0 +1,53 @@
+# RFDK Sample Application - Deadline Spot Event Plugin - Python
+
+## Overview
+[Back to overview](../README.md)
+
+## Instructions
+
+---
+**NOTE**
+
+These instructions assume that your working directory is `examples/deadline/All-In-AWS-Infrastructure-SEP/python/` relative to the root of the AWS-RFDK package.
+
+---
+
+1.  Install the dependencies of the sample app:
+
+    ```bash
+    pip install -r requirements.txt
+    ```
+2. Stage the Docker recipes for `RenderQueue`:
+
+    ```bash
+    # Set this value to the version of RFDK your application targets
+    RFDK_VERSION=<version_of_RFDK>
+
+    # Set this value to the version of AWS Thinkbox Deadline you'd like to deploy to your farm. Deadline 10.1.9 and up are supported.
+    RFDK_DEADLINE_VERSION=<version_of_deadline>
+
+    npx --package=aws-rfdk@${RFDK_VERSION} stage-deadline \
```

Comment:
There's a newer form for `stage-deadline` that doesn't require providing the full S3 URLs. Would probably be better to use that.